src/main/resources/base/console_backend.capnp

```diff
@@ -28,7 +28,7 @@ struct LogLevelFront {
 }
 
 enum LogLevel {
-    error @0;
```

Comment:
fixes
```rust
    error: ambiguous associated item
       --> /Users/adriankong/IdeaProjects/v/swift-toolbox/target/x86_64-apple-darwin/debug/build/console-backend-0f9e7bb137013945/out/console_backend_capnp.rs:831:59
        |
    831 |   fn try_from(value: u16) -> ::core::result::Result<Self, Self::Error> {
        |                                                           ^^^^^^^^^^^ help: use fully-qualified syntax: `<console_backend_capnp::LogLevel as TryFrom>::Error`
        |
        = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
        = note: for more information, see issue #57644 <https://github.com/rust-lang/rust/issues/57644>
    note: `Error` could refer to the variant defined here
       --> /Users/adriankong/IdeaProjects/v/swift-toolbox/target/x86_64-apple-darwin/debug/build/console-backend-0f9e7bb137013945/out/console_backend_capnp.rs:823:3
        |
    823 |   Error = 0,
        |   ^^^^^
    note: `Error` could also refer to the associated type defined here
       --> /Users/adriankong/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/convert/mod.rs:648:5
        |
    648 |     type Error;
        |     ^^^^^^^^^^
        = note: `#[deny(ambiguous_associated_items)]` on by default
    
    error: could not compile `console-backend` due to previous error
```

fixes the compilation issue but probably better to fix this entirely. i.e. seems like in `log_panel.rs` `entry.set_line()` doesnt set the level.

broken into seperate ticket https://swift-nav.atlassian.net/browse/CPP-884 to look into more